device/pcie/pci_device.cpp

```diff
@@ -23,6 +23,47 @@
 #include "common/assert.hpp"
 #include "common/logger.hpp"
 
+static const uint16_t GS_PCIE_DEVICE_ID = 0xfaca;
+static const uint16_t WH_PCIE_DEVICE_ID = 0x401e;
+static const uint16_t BH_PCIE_DEVICE_ID = 0xb140;
+
+// TODO: we'll have to rethink this when KMD takes control of the inbound PCIe
+// TLB windows and there is no longer a pre-defined WC/UC split.
+static const uint32_t GS_BAR0_WC_MAPPING_SIZE = (156<<20) + (10<<21) + (18<<24);
+static const uint32_t BH_BAR0_WC_MAPPING_SIZE = 188<<21; // Defines the address for WC region. addresses 0 to BH_BAR0_WC_MAPPING_SIZE are in WC, above that are UC
```

Comment:
I know this is not your addition, but I think it would be nice to be consistent with comments of the variables. Keep the comments above the variable rather than in the same line. Like I said, change it if you want, if not maybe we can do it for eveyrthing in one PR

Reply:
Fixed